<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ntrip_qtClass</class>
 <widget class="QMainWindow" name="ntrip_qtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ntrip_qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="TabPage">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ClientTab">
       <attribute name="title">
        <string>客户端</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>服务端Ip：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="clientIpEdit">
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>服务端端口：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="clientPortEdit">
            <property name="text">
             <string>2101</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clientConnect">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clientDisconnect">
            <property name="text">
             <string>断开</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>接受数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="clientReceiveText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>发送数据：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="channelSelect">
            <item>
             <property name="text">
              <string>SF01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SF02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SF03</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="clientSendText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="clientSend">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ServerTab">
       <attribute name="title">
        <string>服务端</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>接受数据：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="serverReceiveText">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>发送数据：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="serverSendText">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QCheckBox" name="ForwardCheck">
              <property name="text">
               <string>转发</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ForwardIpEdit">
              <property name="text">
               <string>104.42.214.164</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="severSend">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>服务端端口：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="serverPortEdit">
              <property name="text">
               <string>2101</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="serverListen">
              <property name="text">
               <string>监听</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="serverClose">
              <property name="text">
               <string>关闭</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>客户端列表：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="onlineUserList"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ntrip_qt.qrc"/>
 </resources>
 <connections/>
</ui>
